axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/BlockingQueryResponseProcessingTask.java

```diff
@@ -38,31 +37,34 @@
  * @author Stefan Dragisic
  * @since 4.6.0
  */
-class BlockingQueryResponseProcessingTask<R> implements PrioritizedRunnable {
+class BlockingQueryResponseProcessingTask<R> implements Runnable {
```

Comment:
I realize the name hasn't changed, but I don't see what's "Blocking" about this implementation.

Reply:
That's something we should check with @m1l4n54v1c and @schananas as, honestly, I don't recall either.

Reply:
Gotcha. That makes sense